frontend/src/metabase/entities/questions.js

```diff
@@ -16,11 +17,21 @@ import { canonicalCollectionId } from "metabase/collections/utils";
 
 import forms from "./questions/forms";
 
+const listQuestions = GET("/api/card");
+const listModelsForDatabase = GET("/api/database/:dbId/models");
+
 const Questions = createEntity({
   name: "questions",
   nameOne: "question",
   path: "/api/card",
 
+  api: {
+    list: async (params, ...args) =>
+      params.f === "database-models"
```

Comment:
We don't use such `f` pattern anywhere. Would it make sense to check for `dbId` and/or some `model`/`dataset` flag? This also depends on whether we will use 2 endpoints or just 1.

Reply:
Good question, I also thought about changing `f` but left it because it did seem like a pattern - we already use `f` already for Question queries with the "/api/card" endpoint. e.g. [here](https://github.com/metabase/metabase/blob/ca81479799f93f3f307445e212c5a34e6a2d119d/frontend/src/metabase/models/components/ModelDetailPage/ModelUsageDetails/ModelUsageDetails.tsx#L77) and [here](https://github.com/metabase/metabase/blob/ca81479799f93f3f307445e212c5a34e6a2d119d/frontend/src/metabase/dashboard/components/AddSeriesModal/AddSeriesModal.jsx#L242).

But I think `model=true` makes more sense as it's more obvious to the reader what it does. I'll update the new endpoint.